src/cloud/benchflow/sender/sender.go

```diff
@@ -204,6 +229,7 @@ func launchAnalyserScript(trialID string, totalTrials int, req string) {
 func main() {
 	kafkaIp = os.Getenv("KAFKA_IP")
 	kafkaPort = os.Getenv("KAFKA_PORT")
+	kafkaGroup = os.Getenv("KAFKA_GROUP")
```

Comment:
Why is this an ENV variable? According to my experience each topic should have a different GroupID since the consumer you are instantiating are different for each of the topic.